src/crab/fwd_analyzer.cpp

```
@@ -28,20 +28,32 @@ class InterleavedFwdFixpointIterator final {
     /// Used to skip the analysis until _entry is found
     bool _skip{true};
 
+    bool has_error(const Label& node) const { return _inv.at(node).error.has_value(); }
```

Clang (14.0.6)

[warning] function 'has_error' should be marked [[nodiscard]]

(modernize-use-nodiscard)